tensorboard/examples/plugins/example_raw_scalars/tensorboard_plugin_example_raw_scalars/static/model.js

```diff
@@ -0,0 +1,84 @@
+// Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ==============================================================================
+
+// Generic example of a service that provides data.
+
+/** @type {?Object<string, !Array<string>>} */
+let runToTagInfo = null;
+
+async function updateRunInfo() {
+  if (!runToTagInfo) {
+    runToTagInfo = await fetchJSON('./tags') || {};
```

Comment:
OK—If this were a real plugin, I’d object to the mutable global state,
but incurring this complexity for the purpose of concision is acceptable
to me.

Reply:
Ack, there is some balance between complexity and good practices.  I'm hoping that concision encourages more folks to feel comfortable wiping out the example and rewriting it in their own style.

Reply:
Good point!